lib/beaker/module_install_helper.rb

```diff
@@ -144,6 +167,22 @@ def get_module_source_directory(search_in)
     end
     module_source_dir
   end
+
+  def forge_host
+    fh = ENV['BEAKER_FORGE_HOST'] || ENV['forge_host']
```

Comment:
thats a valid ENV variable that can be passed to beaker to specify the forge host so I thought it best to check for this also